src/components/FileMetadata/index.tsx

```diff
@@ -28,7 +29,9 @@ const FileMetadataBase = ({ file }: PublicProps) => {
           <React.Fragment>
             <dt>{gettext('Download link')}</dt>
             <dd className={styles.downloadURL}>
-              <a href={file.downloadURL}>{file.filename}</a>
+              <a href={makeApiURL({ url: file.downloadURL })}>
```

Comment:
When I download `popup.js` from http://localhost:3000/en-US/browse/502955/versions/1541786/ I get a blank file. The network panel says it's a 200 but the response is empty there, too. Is this a known issue?

Reply:
Yeah, it looks like you're not the only one: https://github.com/mozilla/addons-server/issues/11118#issuecomment-479914017

Reply:
I don't know if the issue comes from code-manager or the API, so I am currently updating my addons-server local dev env to see if the API replies correctly.

Reply:
This looks like a separate bug because I can reproduce without code-manager (i.e. using httpie only)